src/client/sky.cpp

```diff
@@ -766,3 +624,87 @@ void Sky::update(float time_of_day, float time_brightness,
 			video::SColorf(pointcolor), m_horizon_blend() * 0.25);
 	}
 }
+
+void Sky::draw_sun(video::IVideoDriver *driver, float sunsize, video::SColor suncolor, video::SColor suncolor2, float wicked_time_of_day) {
```

Comment:
Please wrap at 90 columns (with tab size 4) and place the opening curly bracket at the start of a new line.